src/coreclr/jit/stacklevelsetter.cpp

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#ifndef TARGET_WASM
```

Comment:
It may be easier to just exclude the .cpp files in CMakeLists.txt instead of ifdefing out each one end-to-end. (The end-to-end ifdef is probably a fine solution for header files.)

Reply:
I think that's a good idea and will consolidate the removal of files that are not required.